web/themes/new_weather_theme/templates/block/block--weathergov-weather-story.html.twig

```diff
@@ -1,7 +1,15 @@
-<div>
-  <h3>{{ content.title }}</h3>
-  {{ content.body | raw }}
+{% if content.title %}
+<div data-wx-weather-story>
+  <h2>Weather story</h2>
+  <p class="text-gray-50">Last updated:
+    <weather-timestamp data-utc="{{ content.updated.utc }}" data-date-format="M d Y T">
+      {{ content.updated.formatted }}
+    </weather-timestamp>
+  </p>
   {% if content.image %}
-  <img src="{{ file_url(content.image.uri) }}" alt="{{ content.image.alt | replace({'"': "&quot;"}) }}">
+  <img class="margin-left-neg-2 tablet:margin-left-0" src="{{ file_url(content.image.uri) }}" alt="{{ content.image.alt | replace({'"': "&quot;"}) }}">
   {% endif %}
+  <h3>{{ content.title }}</h3>
+  {{ content.body | raw }}
```

Comment:
Not sure if we can add a class on this, but the default line-height is a little tight. If possible, could we add `line-height-sans-4` here or by some other means give the paragraph a 1.5 line height (USWDS token value 4).